custom/icds_reports/templates/icds_reports/icds_app/mobile/navigation.directive.html

```diff
@@ -116,10 +116,17 @@
                       Lactating Mothers enrolled for Anganwadi Services</a>
                 </div>
                 <div class="navigation-font pl-3 pr-3 mb-0"
+                     ng-if="haveAccessToFeatures==false"
                      ng-class="{'active-mobile-link': $location.path().indexOf('/demographics/adolescent_girls') !== -1}">
                     <a href="{$ goToStep('demographics/adolescent_girls', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
                       Adolescent Girls enrolled for Anganwadi Services</a>
                 </div>
+              <div class="navigation-font pl-3 pr-3 mb-0"
+                     ng-if="toShowOnNavigation()"
+                     ng-class="{'active-mobile-link': $location.path().indexOf('/demographics/adolescent_girls') !== -1}">
+                    <a href="{$ goToStep('demographics/adolescent_girls', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
+                      Out of school Adolescent girls(11-14 years)</a>
+                </div>
```

Comment:
thanks. very annoying this has to be done in two places. we are working on combining them. cc @saiprakash-v